ddd-koans/Entity/Identifier/src/Task.kt

```diff
@@ -0,0 +1,17 @@
+interface Entity {
+    fun id(): Any
+}
+
+class Bar(
+    private val ordinalNumber: OrdinalNumber,
```

Comment:
I would just name the property `number`.

Reply:
I would go for ordinal, as the number is too broad. [Cambrdige dict](https://dictionary.cambridge.org/pl/dictionary/english/ordinal)